Jenkinsfile.update-api-sdk-php

```diff
@@ -0,0 +1,15 @@
+elifeUpdatePipeline(
+    { commit ->
+        lock('search--ci') {
+            builderDeployRevision 'search--ci', commit
+            builderCmd "search--ci", "composer update elife/api elife/api-client elife/api-sdk --no-interaction", "/srv/search"
```

Comment:
@thewilkybarkid is the list of dependencies to update right? Don't want locks on the other ones keeping api-sdk from being updated